cornell-dti/course-plan
# Add more possible foreign langs in requirements

### Summary 

This pull request adds missing foreign languages codes to the list we have in requirements

![image](https://user-images.githubusercontent.com/25535093/116423603-d6b31080-a80e-11eb-8d20-4db7c19195b6.png)

### Test Plan 

1. Confirm that the list of course codes match the list here: https://lrc.cornell.edu/languages-cornell
2. Try to add a course with one of the new subjects to a requirement that uses the foreign language checker. For example, try to add an ASL course to fulfill the A&S foreign language requirement.

### Notes 

1. I was planning on just adding ASL as a small bug fix (because someone pointed out to me that they could not count their foreign language courses in ASL), but ended up consulting the above list and adding a bunch more. Please let me know if we don't want all of these added for some reason!
2. Some of the course subjects on that website are specifically marked with a hollow dot as not applying to the A&S Foreign Language requirement. Do we want to special case that in some way, or continue to be as open as possible?
